src/ui-todo.c

```diff
@@ -64,10 +65,12 @@ void ui_todo_add(void)
 	status_mesg(mesg, "");
 	if (getstring(win[STA].p, todo_input, BUFSIZ, 0, 1) ==
 	    GETSTRING_VALID) {
-		while ((ch < '0') || (ch > '9')) {
+		do {
 			status_mesg(mesg_id, "");
-			ch = keys_wgetch(win[KEY].p);
-		}
+			if ((ch = keys_wgetch(win[KEY].p)) == 10) {
```

Comment:
Please use `RETURN` instead of the hardcoded `10` here. Looks fine otherwise! Thanks for contributing!